### PR TITLE
Update trigger creation steps to match new UI

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

- https://jira.mongodb.org/browse/DOCSP-28955
- https://jira.mongodb.org/browse/DOCSP-28956

### Staged Changes

- [Database Triggers](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-28955/triggers/database-triggers/)
- [Scheduled Triggers](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-28955/triggers/scheduled-triggers/)

### CODE REVIEW

File: source/triggers/database-triggers.txt [hunk 1]
```diff
@@ -166,17 +193,17 @@ Database Triggers have the following configuration options:
...
           Regardless of this setting:
           
-          - ``INSERT`` and ``REPLACE`` events always include the
+          - **Insert** and **Replace** events always include the
             ``fullDocument`` field.
           
-          - ``DELETE`` events never include the ``fullDocument`` field.
+          - **Delete** events never include the ``fullDocument`` field.
 
    * - :guilabel:`Document Preimage`
```
Comment (cbullinger) [inline source/triggers/database-triggers.txt:201]: My field's disabled with a note "You must be using an Atlas Cluster running MongoDB 4.4+ and a non-sharded cluster to use this feature." Do we mention the reqs for this feature anywhere?

File: source/triggers/database-triggers.txt [hunk 2]
```diff
@@ -113,51 +113,78 @@ Database Triggers have the following configuration options:
...
           Data Explorer fully replace the previous document. As a result,
-          update operations from these clients will generate ``REPLACE``
-          change events rather than ``UPDATE`` events.
+          update operations from these clients will generate **Replace**
+          change events rather than **Update** events.
 
    * - :guilabel:`Full Document`
 
-     - If enabled, ``UPDATE`` change events include the latest
+     - If enabled, Update change events include the latest
```
Comment (cbullinger) [inline source/triggers/database-triggers.txt:187]:
```suggestion
     - If enabled, **Update** change events include the latest
```

**Review by cbullinger — APPROVED** [review]
A couple comments but overall LGTM

File: source/triggers/database-triggers.txt [hunk 3]
```diff
@@ -113,51 +113,78 @@ Database Triggers have the following configuration options:
...
-     - Required. The type of the Trigger. Set this value to ``DATABASE`` for database Triggers
+     - The type of the Trigger. Set this value to :guilabel:`Database` for 
+       database Triggers
 
    * - :guilabel:`Name`
 
-     - Required. The name of the Trigger.
+     - The name of the Trigger.
+      
+   * - | :guilabel:`Enabled`
```
[the remark this answers is not in the document]
Comment (MongoCaleb) [reply source/triggers/database-triggers.txt:123]: Yep. It is "Enabled" in the UI and "disabled" in the json!

**Pull request merged**